pkg/ingester/ingester.go

```diff
@@ -111,6 +112,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.DurationVar(&cfg.MaxChunkAge, "ingester.max-chunk-age", 12*time.Hour, "Maximum chunk age before flushing.")
 	f.DurationVar(&cfg.ChunkAgeJitter, "ingester.chunk-age-jitter", 20*time.Minute, "Range of time to subtract from MaxChunkAge to spread out flushes")
 	f.BoolVar(&cfg.SpreadFlushes, "ingester.spread-flushes", false, "If true, spread series flushes across the whole period of MaxChunkAge")
+	f.IntVar(&cfg.MinChunkLength, "ingester.min-chunk-length", 0, "Minimum number of samples in a chunk to flush it to the store. Use with care, if chunks are less than this size they will be discarded.")
```

Comment:
This is now slightly inaccurate - a chunk can be less than this size and not discarded if it adjoins another chunk in the same series. Also if the ingester goes into 'flush immediately' mode it doesn't apply.

Though I don't suggest you fit all this into the CLI help.

Reply:
I added the word "idle" to the description to make it a bit more accurate.